Lib/test/test_clinic.py

```diff
@@ -3761,19 +3762,19 @@ def test_normalize_snippet(self):
                 actual = clinic.normalize_snippet(snippet, indent=indent)
                 self.assertEqual(actual, expected)
 
-    def test_quoted_for_c_string(self):
+    def test_escaped_docstring(self):
         dataset = (
             # input,    expected
-            (r"abc",    r"abc"),
-            (r"\abc",   r"\\abc"),
-            (r"\a\bc",  r"\\a\\bc"),
-            (r"\a\\bc", r"\\a\\\\bc"),
-            (r'"abc"',  r'\"abc\"'),
-            (r"'a'",    r"\'a\'"),
+            (r"abc",    r'"abc"'),
+            (r"\abc",   r'"\\abc"'),
+            (r"\a\bc",  r'"\\a\\bc"'),
+            (r"\a\\bc", r'"\\a\\\\bc"'),
+            (r'"abc"',  r'"\"abc\""'),
+            (r"'a'",    r'"\'a\'"'),
```

Comment:
What's going on with the extra quotes being added here? Not sure I understand

Reply:
The original test was for the `quoted_for_c_string()` function. Now, this is abstracted away in `Tools/clinic/libclinic/formatting.py` (it is just a helper for `docstring_for_c_string()`. So, now we're testing `docstring_for_c_string` instead, which means we now test the final rendered C string literal, instead of what's inside the double quotes of that C string literal.

Reply:
Ahh, thanks!